src/generated/resources/reports/registry_order.json

```diff
@@ -2,6 +2,7 @@
   "order": [
     "minecraft:attribute",
     "minecraft:data_component_type",
+    "minecraft:armor_material",
```

Comment:
Why these unneeded changes?

Reply:
it keeps generating them when I run data for some reason